subsys/net/ip/l2/ieee802154/ieee802154.c

```diff
@@ -29,6 +29,7 @@
 #include "ieee802154_frame.h"
```

Comment:
The commit message is not proper English, could you change to something like:

```
net/ieee802154: Apply hw filters when applicable

If the hw supports filtering, then apply the ieee address, short address or PAN ID filters.
```

Reply:
ok